editoast/src/views/v2/train_schedule/projection.rs

```diff
@@ -308,15 +315,17 @@ struct TrainSimulationDetails {
 async fn compute_batch_signal_updates<'a>(
     core: Arc<CoreClient>,
     infra: &Infra,
-    track_section_ranges: &'a Vec<TrackRange>,
-    routes: &'a Vec<Identifier>,
+    path_track_ranges: &'a Vec<TrackRange>,
+    path_routes: &'a Vec<Identifier>,
+    path_blocks: &'a Vec<Identifier>,
     trains_details: &'a HashMap<i64, TrainSimulationDetails>,
 ) -> Result<HashMap<i64, Vec<SignalUpdate>>> {
     let request = SignalUpdatesRequest {
         infra: infra.id,
         expected_version: infra.version.clone(),
-        track_section_ranges,
-        routes,
+        track_section_ranges: path_track_ranges,
```

Comment:
You change the naming of `track_section_ranges` for `path_track_ranges`, probably for a better understanding, but you keep the old one here;
it's not mandatory to change it

Reply:
Changing the name here means changing the interface between core and editoast.
`SignalUpdatesRequest` contains the following fields:
- `track_section_ranges`
- routes
- blocks

We've chosen these names because they correspond to the output of the pathfinding endpoint.

Reply:
Yeah, I was speaking about changing the interface because I find the name confusing, but it's not a thing very important to do here !